README.md

```diff
@@ -34,3 +34,26 @@ dataset = app.create_dataset(dataset_id="demo_dataset")
 # execute data upload to Clarifai app dataset
 dataset.upload_dataset(task='visual_segmentation', split="train", dataset_loader='coco_segmentation')
 ```
+
+## Interacting with Inputs
+
+```python
+from clarifai.client.user import User
+app =  User(user_id="user_id").app(app_id="app_id")
+
+# image upload from folder
+image_obj = app.image()
```

Comment:
seems weird to have these `image`, `text` etc. type of functions on the App object. Should they just be class methods on like a `Data` or `Input` object type perhaps? Then app.input() makes sense as that's actually a resource in an app (which technically never was supposed to have a type). image, text, etc. are fields within the data field of the input like input.data.image for example. 

How will multi-modal inputs be supported? 

Do we provide control to use the Data proto directly as there are lots of fields beyond these 4 that need to be support like concepts, regions, frames, metadata, etc. so it'll get complicated fast. 

Not sure the best way to cleanly handle all this.

Reply:
- Makes sense. Missed to add multi-modal support. Thanks for the catch. I'll the change the design slightly and update..

- Regarding the Data proto, we're supporting all of [Data proto](https://github.com/Clarifai/clarifai/blob/71f2239f7e5a23351ed786a92ba724b17350da84/proto/clarifai/api/resources.proto#L596)'s fields as **kwargs** in `input.get_proto()` to create input protos.